src/libstd_unicode/unicode.py

```diff
@@ -446,14 +446,14 @@ def emit_bool_trie(f, name, t_data, is_pub=True):
     f.write("    };\n\n")
 
 def emit_small_bool_trie(f, name, t_data, is_pub=True):
-    last_chunk = max(int(hi / 64) for (lo, hi) in t_data)
+    last_chunk = max(int(hi // 64) for (lo, hi) in t_data)
```

Comment:
Will we ever get non-integer from the source code? All the `int()` calls could be dropped in not.